demos/sjtwo/system_clock/source/main.cpp

```diff
@@ -2,33 +2,89 @@
 #include <cstdint>
 #include <cstdio>
 
+#include "L0_Platform/startup.hpp"
 #include "L1_Peripheral/lpc40xx/pin.hpp"
 #include "L1_Peripheral/lpc40xx/system_controller.hpp"
 #include "utility/log.hpp"
+#include "utility/bit.hpp"
 
 int main()
 {
-  sjsu::lpc40xx::SystemController clock;
-  sjsu::lpc40xx::Pin clock_pin(1, 25);
+  auto & system = sjsu::SystemController::GetPlatformController();
+  auto & config = system.GetClockConfiguration<
+      sjsu::lpc40xx::SystemController::ClockConfiguration>();
+  // Select system clock (use crystal oscillator directly)
+  config.cpu.select =
+      sjsu::lpc40xx::SystemController::CpuClockSelect::kSystemClock;
+  // Make sure to utilize the internal oscillator only.
+  config.system_oscillator =
+      sjsu::lpc40xx::SystemController::OscillatorSource::kIrc;
+  // Make sure PLL0 is enabled and available
+  config.pll[0].enabled  = true;
+  config.pll[0].multiply = 4;
```

Comment:
Might be a good note that the main PLL is configured to output 48 MHz.

Reply:
Fill do.